backend/app/api/routes/schedule.py

```diff
@@ -47,16 +47,30 @@ def create_schedule(
 def update_scheduled_inquiries(
     *, session: SessionDep, scheduled_inquiries: list[int]
 ) -> SchedulePublic:
-    db_schedule = schedule_service.update_scheduled_inquiries(
-        session=session, scheduled_inquiries=scheduled_inquiries
-    )
-    schedule_data = ScheduleInfo.model_validate_json(db_schedule.schedule)
-    scheduled_inquiries = json.loads(db_schedule.scheduled_inquiries)
-    return SchedulePublic(
+    db_schedule = schedule_service.get_schedule(session)
```

Comment:
I'm wondering if this is too much business logic to go into a route file. Should the json load/validate be in the service instead?